application/dash_application/views/sample_swaps.py

```diff
@@ -45,79 +45,74 @@
 }
 
 rename_columns = {
-    COL.LibraryLeft: "QUERY_LIBRARY",
-    COL.LibraryRight: "BEST_MATCH",
     PINERY_COL.StudyTitle: 'PROJECT',
 }
 
 swap = df_manipulation.get_crosscheckfingerprints()
-swap = swap[~swap[COL.LibraryLeft].str.startswith("GSICAPBENCH")]
```

Comment:
Does filtering out GSICAPBENCH and GLCS also happen in qc-etl?

Reply:
GSICAPBENCH is removed for all qc-etl caches (in etl-collectors) and GLCS is removed at the crosscheckfingerprints refiller level. Dashi shouldn't have done this filtering in the first place.